ocfweb/main/templates/main/staff-hours.html

```diff
@@ -27,11 +27,13 @@ <h2>Have questions? Drop by for help from a friendly volunteer staffer!</h2>
                         <div class="content">
                             <ul class="ocf-staffhours-faces">
                                 {%for staffer in staff_hour.staff %}
+                <a href="#" data-toggle="tooltip" data-placement="top" title= {{staffer|get_bio}}>
```

Comment:
where does `get_bios` get its data from? also, make sure to upload the code for that, preferably, in another PR